src/model/entity/enemy_test.go

```diff
@@ -0,0 +1,403 @@
+package entity
+
+import (
+	"testing"
+)
+
+func TestZombieConstructor(t *testing.T) {
```

Comment:
Лучше именовать тест по имени функции, которая непосредственно тестируется:

```suggestion
func TestNewZombie(t *testing.T) {
```

И по другим то же самое.